docs/source/workflows/observe/index.md

```diff
@@ -70,104 +88,26 @@ The `logging` section contains one or more logging providers. Each provider has
 - `console`: Writes logs to the console.
 - `file`: Writes logs to a file.
 
-To see the complete list of configuration fields for each provider, utilize the `aiq info components -t logging` command which will display the configuration fields for each provider. For example:
+### **Tracing Configuration**
 
-```bash
-$ aiq info components -t logging
-                                                    AIQ Toolkit Search Results
-┏━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
-┃ package    ┃ version              ┃ component_type ┃ component_name ┃ description                                               ┃
-┡━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
-│ aiqtoolkit │ 1.2.0.dev15+g2322037 │ logging        │ console        │ A logger to write runtime logs to the console.            │
-│            │                      │                │                │                                                           │
-│            │                      │                │                │   Args:                                                   │
-│            │                      │                │                │     _type (str): The type of the object.                  │
-│            │                      │                │                │     level (str): The logging level of console logger.     │
-├────────────┼──────────────────────┼────────────────┼────────────────┼───────────────────────────────────────────────────────────┤
-│ aiqtoolkit │ 1.2.0.dev15+g2322037 │ logging        │ file           │ A logger to write runtime logs to a file.                 │
-│            │                      │                │                │                                                           │
-│            │                      │                │                │   Args:                                                   │
-│            │                      │                │                │     _type (str): The type of the object.                  │
-│            │                      │                │                │     path (str): The file path to save the logging output. │
-│            │                      │                │                │     level (str): The logging level of file logger.        │
-└────────────┴──────────────────────┴────────────────┴────────────────┴───────────────────────────────────────────────────────────┘
-```
+The `tracing` section contains one or more tracing providers. Each provider has a `_type` and optional configuration fields. The observability system supports multiple concurrent exporters. The following tracing providers are available:
```

Comment:
Drop the `The following tracing providers are available:` part because `Available Tracing Exporters` has it's own heading

```suggestion
The `tracing` section contains one or more tracing providers. Each provider has a `_type` and optional configuration fields. The observability system supports multiple concurrent exporters.
```